### PR TITLE
native: clean up return use

### LINKED ISSUE #70: native: clean up return use in a number of routines
There are a number of routines in native that have naked returns or returns that are unnecessary because of other flow control:

Dgetrf, Dlacpy, Dlaswp, Dlatrs, Dorml2, Dormlq, Dormqr and Dtritri.

### PR DESCRIPTION
**Pull request by kortschak**
Fixes #70.

@btracey Please take a look.

### CODE REVIEW

**Comment by kortschak** [comment]
*ping*

File: native/dgetrf.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ func (impl Implementation) Dgetrf(m, n int, a []float64, lda int, ipiv []int) (o
 		panic(badIpiv)
 	}
 	if m == 0 || n == 0 {
-		return
+		return false
```
Comment (btracey) [inline native/dgetrf.go]: Should this return false? Lapack returns the equivalent of true. I hate these "empty set" issues.

File: native/dtrtri.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ func (impl Implementation) Dtrtri(uplo blas.Uplo, diag blas.Diag, n int, a []flo
 		panic(badDiag)
 	}
 	if n == 0 {
-		return
+		return false
```
Comment (btracey) [inline native/dtrtri.go]: Same question here.

**Comment by btracey** [comment]
LGTM minus the "return false" issues. Probably doesn't really matter.

File: native/dgetrf.go (hunk 1 above)
Comment (kortschak) [inline native/dgetrf.go]: I'm just doing what it was doing before. If you think it should return true, I'm fine with that, but it's not what it was doing prior to this change.

File: native/dtrtri.go (hunk 2 above)
Comment (kortschak) [inline native/dtrtri.go]: Same answer here.

File: native/dgetrf.go (hunk 1 above)
Comment (btracey) [inline native/dgetrf.go]: Good point. I hadn't thought about that. Hasn't caused any issues, so LGTM. If it breaks something we can reassess.

**Pull request merged**